modules/web-discovery-project/tests/integration/utility-regression-test.es

```diff
@@ -0,0 +1,158 @@
+  "https://www.propublica.org/article/new-mexico-foster-care-rtc-teens",
+  "https://www.nbcnews.com/politics/congress/republican-sen-ben-sasse-will-resign-congress-rcna51109",
+  // "https://www.wsj.com/articles/russian-missiles-hit-civilian-targets-in-southeastern-ukraine-11665046368",
+  "https://www.thedailybeast.com/jaguar-queen-on-the-lam-after-cops-use-instagram-to-id-her?source=articles&via=rss",
+  "https://www.telegraph.co.uk/news/2022/10/06/shakespeares-globe-rushes-delete-derogatory-poem-transphobic/",
+  "https://www.motherjones.com/environment/2022/10/north-carolina-outer-banks-sea-level-rise-homes-consumed/",
+  "https://newrepublic.com/article/167979/gretchen-whitmer-2022-reelection-presidential-nominee-2024",
+  "https://news.sky.com/story/ukraine-war-first-time-since-the-cuban-missile-crisis-we-have-a-direct-threat-of-the-use-nuclear-weapons-says-president-biden-12714106",
+  // "https://www.npr.org/2022/10/07/1125588656/comic-how-foraging-restored-my-relationship-with-food",
+  // "https://www.huffpost.com/entry/european-union-agrees-on-price-cap-for-russian-oil-over-ukraine-war_n_633e2379e4b028164531059c",
+  "https://www.politico.com/video/2022/09/28/newsom-says-dems-have-a-messaging-problem-716228",
+  "https://www.theguardian.com/environment/2022/oct/05/drone-footage-orcas-killing-white-shark-south-africa",
+  "https://www.washingtontimes.com/news/2022/oct/7/nobel-peace-prize-to-activists-from-belarus-russia/?utm_source=RSS_Feed&utm_medium=RSS",
+  "https://www.breitbart.com/politics/2022/10/06/democrat-mike-franken-accuser-responds-to-backlash-multiple-women-allege-serial-misconduct/",
+  "https://www.theblaze.com/news/alternatives-to-detention-program-gives-some-illegal-migrants-smartphone-monitoring-devices-and-lets-them-loose-in-the-us-costing-taxpayers-billions",
+  "https://dailycaller.com/2022/10/06/washington-dc-crime-shootings-four-victims-north-capitol-street/",
+  "https://www.csmonitor.com/World/Asia-South-Central/2022/1006/Meet-the-amateur-art-sleuths-helping-bring-back-Asia-s-stolen-heritage?icid=rss",
+  // "https://www.spiegel.de/international/germany/inflation-bankruptcies-and-fears-of-decline-is-this-the-return-of-the-sick-man-of-europe-a-41d914b0-0322-46a8-9087-d118b3f0b399#ref=rss",
+  "https://www.thenation.com/article/society/mahsa-amini-and-the-women-of-iran/",
+  "https://www.democracynow.org/2022/10/5/haiti_demands_pm_resignation_lower_fuel",
+]
+
+export default () => {
+    describe("UtilityRegression tests", () => {
+        const getSuffix = (path = "base") => `/${path}`;
+        const getUrl = (path = "base") => testServer.getBaseUrl(getSuffix(path));
+
+        const WebDiscoveryProject = app.modules["web-discovery-project"].background.webDiscoveryProject;
+        const pipeline = app.modules["webrequest-pipeline"].background;
+        let addPipeline;
+
+        const addCookiesToRequest = (request, response) => {
+          response.modifyHeader("Cookie", "cookie-text")
```

Comment:
Did we need to add specific cookies here for consent popups?

Reply:
removed for now 👍